.github/workflows/cmake.yaml

```diff
@@ -53,6 +56,14 @@ jobs:
           echo " -- install our dependencies"
           sudo apt install -y make cmake openssh-server libgsl-dev libopenmpi-dev libhdf5-dev
 
+      # https://github.com/actions/runner-images/issues/9866#issuecomment-2114583692
+      - name: install GCC 13
+        if: matrix.compiler == 'gcc-13'
+        run: |
+          sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+          sudo apt-get update -y
+          sudo apt-get install gcc-13 g++-13
```

Comment:
FYI, I've had a pretty good experience so far using the [setup-cpp](https://github.com/aminya/setup-cpp) action to install compilers. It's nice that it's cross-platform, handles gcc & clang, and doesn't require me to write any bash commands.

https://github.com/isce-framework/libwhirlwind/blob/5d133ff8419e76b1d649cc0d3050cff90f344358/.github/workflows/ci.yml#L52-L54